Item(by='josefx', descendants=None, kids=[25275055], score=None, time=1606903061, title=None, item_type='comment', url=None, parent=25274747, text='So Go got it right by calling it panic&#x2F;recover instead of throw&#x2F;catch? Should we petition other languages to also change the keywords and correct whatever is wrong with throw&#x2F;catch? Is Pythons raise&#x2F;except safe or did they miss it when they cleaned up the master&#x2F;slave mess?')